index.html

```diff
@@ -0,0 +1,128 @@
+      <div class="main-section-div">
+        <img src="images/better components.png" alt="Small icon">
+        <h6 class="main-section-div-h6"> Better components  </h6>
+        <p class="main-section-div-p">Далеко-далеко за словесными горами в стране гласных и согласных живут рыбные тексты. Необходимыми, вершину оксмокс. </p>
+      </div>
+      <div class="main-section-div">
+        <img src="images/well organised.png" alt="Small icon">
+        <h6 class="main-section-div-h6"> Well organised </h6>
+        <p class="main-section-div-p"> Далеко-далеко за словесными горами в стране гласных и согласных живут рыбные тексты. Необходимыми, вершину оксмокс. </p>
+      </div>
+    </section>
+    <div class="third-div">
+      <div class="left-side">
+        <div class="left-top">
+        <img src="./images/Quote Mark.png" alt="Quote mark">
+        <div>
+          <h1>
+             Real Stories from <br>
+          Real Customers
+          </h1>
+          <p class="common">
+            Get inspired by these stories
+          </p>
+        </div>
+        </div>
+        <div class="left-bottom">
+          <img  class="clr" src="./images/Hubspot Logo Floyd Miles.png" alt="logo">
+          <div>
+            <img src="./images/Quote Mark.png" alt="Quote">
+            <div>
+              <h5>
+                To quickly start my startup landing page design, I was looking for a landing page UI Kit. Landify is one of the best landing page UI kit I have come across. It’s so flexible, well organised and easily editable.
+              </h5>
+              <h3>
+                Floyd Miles
+              </h3>
+              <p>
+                Vice Precident, GoPro
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="right-side">
+        <div class="right-top">
+          <img src="./images/Airbnb Logo jane cooper.png" alt="logo">
+           <div>
+            <img src="./images/Quote Mark.png" alt="quote mark">
+            <div>
+              <p>
+                I used landify and created a landing page for my startup within a week. The Landify UI Kit is simple and highly intuitive, so anyone can use it.
+              </p>
+              <h4>
+                Jane Cooper
+              </h4>
+              <h6>
+                Ceo,Airbnb
+              </h6>
+            </div>
+           </div>
+        </div>
+        <div class="right-bottom">
+           <img src="./images/BookMyShow Logo Kristin watson.png" alt="logo">
+           <div>
+            <img src="./images/Quote Mark.png" alt="quote mark">
+            <div>
+              <p>
+               Landify saved our time in designing my company page
+              </p>
+              <h5>
+                Kristin watson
+              </h5>
+              <h6>
+                Co-Founder, BookMyShow
+              </h6>
+            </div>
+           </div>
+        </div>
+      </div>
+    </div>
+
+
```

Comment:
- Pls restructure all of this. using `<section></section` tag for each section
- And same as the earlier reviews pls, use meaningful class names
- The text should be in English as shown in the design

Reply:
done , please check them if it is okay